tests/masternode/test_set_bet.py

```diff
@@ -0,0 +1,65 @@
+"""
+Provide tests for command line interface's masternode set bet command.
+"""
+import json
+
+from click.testing import CliRunner
+
+from cli.constants import (
+    FAILED_EXIT_FROM_COMMAND_CODE,
+    PASSED_EXIT_FROM_COMMAND_CODE,
+)
+from cli.entrypoint import cli
+from cli.utils import dict_to_pretty_json
+
+
+def test_set_bet(mocker, set_bet_masternode_transaction):
+    """
+    Case: set the masternode betting behaviour.
```

Comment:
`set the masternode betting behaviour` > `set the masternode max betting behaviour`.

Reply:
@dmytrostriletskyi, done.